### PR TITLE
fix: find common titles only on word boundaries

### PR DESCRIPTION
**Pull request by pmuir**
Also, switch to dependabot message format

#### Submitter checklist

- [ ] Change is code complete and matches issue description.
- [ ] Change is covered by existing or new tests.

#### Which issue this PR fixes

fixes #

### CODE REVIEW

File: jx/bdd/tekton/ci.sh [hunk 1]
```diff
@@ -40,5 +40,5 @@ cp ${JX} /usr/bin
 
 echo "running the BDD tests with JX_HOME = $JX_HOME"
 
-jx step bdd --versions-repo https://github.com/jenkins-x/jenkins-x-versions.git --config jx/bdd/tekton/cluster.yaml --gopath /tmp  --git-provider=github --git-username $GH_USERNAME --git-owner $GH_OWNER --git-api-token $GH_CREDS_PSW --default-admin-password $JENKINS_CREDS_PSW --no-delete-app --no-delete-repo --tekton --tests install --tests test-create-spring
+jx step bdd --versions-repo https://github.com/pmuir/jenkins-x-versions.git --config jx/bdd/tekton/cluster.yaml --gopath /tmp  --git-provider=github --git-username $GH_USERNAME --git-owner $GH_OWNER --git-api-token $GH_CREDS_PSW --default-admin-password $JENKINS_CREDS_PSW --no-delete-app --no-delete-repo --tekton --tests install --tests test-create-spring
```
Comment (garethjevans) [inline jx/bdd/tekton/ci.sh]: is this correct?

Comment (pmuir) [reply jx/bdd/tekton/ci.sh]: oops, no

**Pull request merged**